api/views/volunteer.py

```diff
@@ -58,6 +67,61 @@ def summary(self, request: Request, username: str = None) -> Response:
         user = get_object_or_404(BlossomUser, username=username, is_volunteer=True)
         return Response(self.serializer_class(user).data)
 
+    @swagger_auto_schema(
+        operation_summary=(
+            "Retrieve a count of transcriptions for a volunteer per UTC day."
```

Comment:
What does the time_frame option resemble? Is it the number of days included, or the type of aggregation?

Reply:
From what I gather, it is the type of aggregation. I would also reflect this in the summary of operation.

Reply:
It's basically the length of the time frame that is used to aggregate the rate.
If you specify "hour", it will return the hourly transcription rate.

I couldn't think of a better name, do you have any suggestions?

Reply:
I added a parameter description to clarify it a bit.